lang/ghc/Portfile

```diff
@@ -123,11 +125,15 @@ if { [variant_isset "prebuilt"] } {
 
     # add the bootstrap binary and the sphinx-build binary to PATH
     configure.env-append \
-        PATH=$env(PATH):${bootstrap_dir}/bin:${frameworks_dir}/Python.framework/Versions/${python3_version}/bin
+        GHC=${bootstrap_dir}/bin/ghc \
+        "PATH=/usr/bin:$env(PATH):${bootstrap_dir}/bin:${frameworks_dir}/Python.framework/Versions/${python3_version}/bin"
+
+    # Fix Xcode 11 issue; see https://trac.macports.org/ticket/59467
+    compiler.blacklist-append clang
```

Comment:
Steve, are you sure that you need to blacklist clang? The issue in ticket 59467 was all about a definition that was not to be used on Catalina <https://gitlab.haskell.org/ghc/ghc/merge_requests/2190/diffs?commit_id=c6fca740a708ad9db5fdb595ad06f727289cdc07> and had nothing to do with Xcode 11...

Reply:
Thanks! Removed.